test/integration/concierge_whoami_test.go

```diff
@@ -89,6 +89,7 @@ func TestWhoAmI_ServiceAccount_Legacy_Parallel(t *testing.T) {
 	}, metav1.CreateOptions{})
 	require.NoError(t, err)
 
+	// TODO: What is this used for?
```

Comment:
What about this TODO?

Reply:
This usage of service account tokens is not relevant to this PR. Removed the TODO.